plugins/asus-hid/fu-asus-hid-plugin.c

```diff
@@ -28,6 +28,7 @@ fu_asus_hid_plugin_constructed(GObject *obj)
 	FuContext *ctx = fu_plugin_get_context(plugin);
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_ASUS_HID_DEVICE);
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_ASUS_HID_CHILD_DEVICE); /* coverage */
+	fu_plugin_set_device_gtype_default(plugin, FU_TYPE_ASUS_HID_DEVICE);
```

Comment:
if you do this, you don't need `fu_plugin_add_device_gtype(plugin, FU_TYPE_ASUS_HID_DEVICE);`